src/pages/Checkout/CheckoutPage.vue

```diff
@@ -493,6 +498,21 @@ export default {
 		// TODO: Implement check against contentful setting
 		// to signify if holiday mode is enabled
 
+		// VUE-1725 Async Checkout Mutation Rollout
+		const asyncCheckoutExp = this.apollo.readFragment({
+			id: `Experiment:${ASYNC_CHECKOUT_EXP}`,
+			fragment: experimentVersionFragment,
+		}) || {};
+
+		this.asyncCheckoutExpVersion = asyncCheckoutExp?.version;
```

Comment:
This could instead store `asyncCheckoutExp?.version === 'b'`

Reply:
True I could convert that to `asyncCheckoutActive` instead...